assets/services/assets-api/src/utils/number.ts

```diff
@@ -95,3 +95,30 @@ export const addDecimalsAndConvertToHex = (number, decimals): string => {
     throw new Error(`addDecimalsAndConvertToHex --> ${error.message}`);
   }
 };
+
+/**
+ * [Add Numbers By Converting Into BigNumber]
+ *
+ * The purpose of this function is to add numbers after converting
+ * them to BigNumber and then return the Number
+ *
+ * This function is used everytime we add two numbers.
+ *
+ */
+export const addNumbersByConvertingIntoBigNumber = (number1, number2): number => {
+  try {
+    const number1Bn = new BigNumber(number1);
+    const number2Bn = new BigNumber(number2);
+    const sumBn = number1Bn.plus(number2Bn);
+    return parseFloat(sumBn.toString());
+  } catch (error) {
+    ErrorService.logAndThrowFunctionError(
+        error,
+        'adding Numbers by converting to BigNumber first',
```

Comment:
A failing message should be placed here. Also will be nice to print also the numbers in message:
Failed to add numbers by converting to BigNumber first: number1, number2

```suggestion
        'Failed to add numbers by converting to BigNumber first',
```